### PR TITLE
Added a note about BSDs

### PR DESCRIPTION
**Pull request by mzs114**
Related to issue #15

### CODE REVIEW

**Comment by lmanul** [comment]
Why the need to link to BSD OSes since the name and URL of the website specifically mention Linux?

**Comment by mzs114** [comment]
It is about making an end user aware of the choice.
With the link we are leaving a subtle hint saying that user has a choice, for his curiosity, or learning, exploring, etc.

Linux by itself is just the kernel, not the complete OS, GNU software makes it an OS. 
Most of the features listed are actually provided by software which runs on top of it like  KDE, GNOME, applications etc. 
Linux is not providing these.

BSDs are considered to be complete OSs, so why not BSDs, which are complete OSs in their own right?

File: index.php [hunk 1]
```diff
@@ -23,7 +23,8 @@
 <a href="http://github.com/lmanul/whylinuxisbetter">GitHub</a> and come contribute!</p>
 
 <p>Have a look at <a href="switch/try/oneday/index.php">One day with Linux (a screenshot tour)</a>!</p>
-
+<p>Note: The following features are not specific to just Linux,
```
Comment (lmanul) [inline index.php]: I don't understand what warrants a mention to BSDs (especially at the top of the front page) when the site is specifically about Linux. Sounds like a desperate "hey, I exist too, look at meeeee" call :-)

Comment (mzs114) [inline index.php]: I disagree, its about a simple note to the reader that these features are not specific to GNU/Linux and there are other alternatives(BSD, Solaris, etc).

I think my above post justifies why this was a correct approach.

**Pull request closed without merging**